frontend/src/components/RideModal/RideModal.tsx

```diff
@@ -66,16 +102,12 @@ const RideModal = () => {
           formData={formData}
           onSubmit={saveDataThen(goNextPage)}
         />
-        <DriverPage
-          formData={formData}
-          onBack={goPrevPage}
-          onSubmit={saveDataThen(goNextPage)} />
```

Comment:
we only want the driverpage to be removed when editing a ride, but this removes it for creating a ride too